src/freenas/etc/systemd/system/kdump-tools-dump.service.d/override.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
```

Comment:
Why two ExecStart is necessary?

Reply:
First one is to clear the default `ExecStart` action, if it's not done it will execute it's core dump saving script